pkgs/development/tools/godot/default.nix

```diff
@@ -1,108 +1,136 @@
-{ stdenv
```

Comment:
This should have `lib` and `stdenv` first.

Reply:
I just sorted them alphabetically.  I'll move these two back to the top though.

Reply:
You can sort them however you'd like after those two -- it's just convention (for whatever reason) to have those two first.